traffic-participants.rst

```diff
@@ -0,0 +1,96 @@
+Modeling traffic participants
+==============================
+
+Traffic participants
+--------------------
+
+A traffic participant is an element of the simulated world, which can change its position and orientation during simulation time.
+A traffic participant represents one of the following:
+
+- A living being.
+- A means of transportation for living beings
+- A means of transportation for goods
+- Any other movable object that may travel the road network
+
+Pedestrians and animals are examples of traffic participants that are living beings.
+Vehicles are examples of traffic participants that are means of transportation.
+Therefore, the ego vehicle is also a traffic participant.
+
+The following figure shows the interface of a traffic participant.
+
+.. image:: _static/images/osi-traffic-participant-principle.png
+   :align: center
+   :alt: Traffic participant interfaces
+
+With every simulation step, an OSI traffic participant model receives ground truth data from the environment around itself, the sensor view.
+A traffic participant can output its own perceivable state, the traffic update.
+Traffic commands influence the behavior of the traffic participant model.
+Traffic commands allow event-based communication towards the traffic participant, for example, at certain simulation steps.
+OSI only specifies the interface of the traffic participant.
+OSI is agnostic to the architecture of the simulation environment and the traffic participant.
+
+
+Use cases for traffic participants
+----------------------------------
```

Comment:
After the discussion today, I would in all pictures substitute the "Simulator" block with "Environment simulation". Additionally, is it possible to better illustrate scenario and map as files? I think OpenDRIVE and OpenSCENARIO should be marked as examples for those files ...

Reply:
- [ ] Replace "Simulator" with "environment simulation" in all images.

Reply:
Sorry it was not very precise what I wrote. I think with our discussion last week, this box should ONLY include "Environment simulation". Otherwise it is just confusing. 
What about the other points:

- Additionally, is it possible to better illustrate scenario and map as files? I think OpenDRIVE and OpenSCENARIO should be marked as examples for those files.

- Also the models without current OSI interfaces should be greyed out I guess, consistent to the vision document.

Reply:
- [ ] Use only "Environment simulation" in graphics.
- [ ] Mark ODR and OSC as examples.
- [ ] Grey-out non-supported interfaces.

Reply:
Thanks, in my opinion it looks ok now.

Reply:
One question regarding `osi-traffic-participant-use-case-5.png`: The arrow from _environment simulation_ to _Mock-up (DiL)_ is currently not labeled. I was not sure whether this arrow is representative of any OSI message like TrafficCommand or whether it is user-defined. In the first case, I would a a label to the arrow. In the latter case, I would grey it out. Is this arrow representative of an OSI message and if yes, which?

Reply:
I think it is a good idea to grey it out for the 3.3 release. In the future it is more meant like this: There will be an OSI visualization interface (e.g. GroundTruth) that can be input to a renderer /graphics engine. That is what the driver in the loop looks at. We might then show another component of a renderer in the picture which consumes OSI sensor view or groundtruth. I would see it out of scope for 3.3 and would say with the greyed out arrow it should be ok for now. @ThomasNaderBMW feel free to comment if you have another opinion.